docs/src/views/FocusZone.tsx

```diff
@@ -0,0 +1,366 @@
+        listeners applied there.
+        <CodeSnippet
+          label="html"
+          value={`
+            <div>---> FocusZone's attributes/events listeners applied here            
+              <ul role="menu" class="ui-menu">
+                <li class="ui-menu__item ..." role="presentation">
+                </li>
+                <li class="ui-menu__item ..." role="presentation">
+                </li>
+              </ul>
+            </div>
+          `}
+        />
+      </li>
+    </ul>
+    <Header as="h3">Props</Header>
+    <p>To FocusZone next props can be applied:</p>
+    <ul>
+      <li>
+        <b>as</b> - Element type the root element will use. Default is "div". Only applies to
+        FocusZone's container in {code('Wrap')} mode.
+        <p>Type: {code('React.ReactType')}</p>
+      </li>
+      <li>
+        <b>className</b> - Additional class name to provide on the root element, in addition to the
+        ms-FocusZone class.
+        <p>Type: {code('string')}</p>
+      </li>
+      <li>
+        <b>direction</b> - Defines which arrows to react to.
+        <p>Type: {code('FocusZoneDirection')}, enum with next options:</p>
+        <ul>
+          <li>
+            <i>horizontal</i> - navigation between items can be made using left/right arrow keys.
+            <p>See an example of {link('horizontal menu', '/components/menu#types-menu')}.</p>
+          </li>
+          <li>
+            <i>vertical</i> - navigation between items can be made using up/down arrow keys.
+            <p>
+              See an example of{' '}
+              {link('vertical menu', '/components/menu#variations-vertical-pointing')}.
+            </p>
+          </li>
+          <p>
+            Vertical and horizontal menu share same accessibility behavior and direction is defined
+            by prop "vertical":
+          </p>
+          <CodeSnippet
+            label="menuBehavior.ts"
+            value={`
+              const menuBehavior: Accessibility = (props: any) => ({
+                //...
+                focusZone: {
+                  mode: FocusZoneMode.Embed,
+                  props: {
+                    direction: props.vertical ?
+                    FocusZoneDirection.vertical :
+                    FocusZoneDirection.horizontal,
+                    //...
+                  },
+                },
+              })
+              `}
+          />
+          <li>
+            <i>bidirectional</i> - navigation between items can be made using all arrow keys.{' '}
+            <b>It is set by default.</b>
+            <p>See an example of {link('tab list', '/components/menu#usages-tab-list')}.</p>
+          </li>
+        </ul>
+      </li>
+      <li>
+        <b>defaultTabbableElement</b> - Function which uses root element as parameter to return the
+        initial active element. For example, when there is a chat with a bottom-up approach, it is
```

Comment:
... approach, the last chat message needs to be tabbable (active), not the first default one.